frontend/public/components/catalog-items.jsx

```diff
@@ -247,7 +254,7 @@ export class CatalogTileViewPage extends React.Component {
           <CatalogTile
             id={uid}
             key={uid}
-            href={href}
+            onClick={() => this.onItemClick(href)}
```

Comment:
Ugh sorry, I just realized you need to pass a parameter, so we need fat arrow here anyway. My fault.

We should undo the bind in the constructor since it doesn't help :(

Reply:
Sorry for all the noise

Reply:
Done. Thanks.